apps/docs/src/app/platform/component-docs/platform-forms/checkbox/platform-checkbox-examples/platform-checkbox-error-handling.component.html

```diff
@@ -12,8 +12,8 @@ <h3>Checkbox Error Handling</h3>
         </fdp-checkbox>
     </fdp-form-field>
 
-    <fdp-form-field [id]="'button-1'" [rank]="20" zone="zRight">
-        <fdp-button [buttonType]="'emphasized'" (click)="onSubmit()">Submit</fdp-button>
+    <fdp-form-field id="button-1" [rank]="20" zone="zRight">
+        <fdp-button buttonType="emphasized" (click)="onSubmit()" label="Submit">Submit</fdp-button>
```

Comment:
Minor thing. Removing Submit inside tag is missed here. Not a issue.

Reply:
oops, i thought I had removed it.. will remove it, just in case it has unintended results later on. Thanks for pointing it out!